Update Localisation Guidelines (#3153)
alan-turing-institute/the-turing-way: book/website/community-handbook/translation/translation-workflow.md

We are members of The Turing Way community with different motivations to localise the content to different languages.
Feel free to join the `#translation` channel in [_The Turing Way_ slack](https://theturingway.slack.com).

> You will also have to create a Crowdin account but more on that [in the next section](#ch-translation-localisation-journey).

Reviewer: ```suggestion
You will also have to create a Crowdin account but more on that {ref}`in the next section<ch-translation-localisation-journey>`
```

We are members of The Turing Way community with different motivations to localise the content to different languages.
Feel free to join the `#translation` channel in [_The Turing Way_ slack](https://theturingway.slack.com).

> You will also have to create a Crowdin account but more on that [in the next section](#ch-translation-getting-started).

Reviewer: ```suggestion
You will also have to create a Crowdin account but more on that [in the next section](#ch-translation-localisation-journey).
```
Looks like this chapter was renamed?

## Asynchronous translation

> Everyone interested in participating in a translation effort should be able to work asynchronously, so don't worry if you cannot join the bi-weekly calls. More info can be found in the next chapter about [our localisation workflow](https://deploy-preview-2202--the-turing-way.netlify.app/community-handbook/translation/getting-started.html).

Reviewer: ```suggestion
Everyone interested in participating in a translation effort should be able to work asynchronously, so don't worry if you cannot join the fortnightly calls. More info can be found in the next chapter about {ref}`our localisation workflow<ch-translation-localisation-journey>`.
```

Every team has a repository with these guidelines in the GitHub organisation. 
Check examples for
[Portuguese](https://github.com/TWTranslation/Portuguese_specific_translation_guidelines), [Arabic](https://github.com/TWTranslation/Arabic-specific-Translation-rules), and [Turkish](https://github.com/TWTranslation/Turkish-specific-Translation-rules).

> We have also set up a public page for _The Turing Way_ in [Crowdin](https://turingway.crowdin.com/turing-way) with a README file summarising some of the translation guidelines. Make sure you read these guidelines before you start translating for the first time.

Reviewer: ```suggestion
We have also set up a public page for _The Turing Way_ in [Crowdin](https://turingway.crowdin.com/turing-way) with a README file summarising some of the translation guidelines. 
Make sure you read these guidelines before you start translating for the first time.
```
https://the-turing-way.netlify.app/community-handbook/style.html#write-each-sentence-in-a-new-line-line-breaks

In each co-working or external session, prioritise those chapters that are outdated or close to completion. 

> Check out our [guidelines](ch-translation-localisation-journey) for a list of the priority list.

Reviewer: ```suggestion
Check out our {ref}`guidelines<ch-translation-localisation-journey>` for a list of the priority list.
```
